app/controllers/UserDataController.scala

```diff
@@ -83,16 +96,16 @@ class UserDataController(frontsApi: FrontsApi, dynamoClient: AmazonDynamoDB, val
   def putFeatureSwitch() = APIAuthAction { request =>
     val maybeFeatureSwitch: Option[FeatureSwitch] = request.body.asJson.flatMap(
       _.asOpt[FeatureSwitch])
-    val maybeUserData: Option[UserData] = Scanamo.exec(dynamoClient)(
-      userDataTable.get('email -> request.user.email)).flatMap(_.right.toOption)
+    val maybeUserData: Option[UserData] = Scanamo(dynamoClient).exec(
+      userDataTable.get("email" === request.user.email)).flatMap(_.toOption)
```

Comment:
Is this safe? IIUC this can be `None` if there is no record or if scanamo blows up due to permissions or other failure. To be fair it was already like this, so perhaps not something to fix now....